test/integration/test-metrics.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
Here's what it looks like when you run the script locally:
<img width="843" alt="2023-12-11-18-42-03" src="https://github.com/warm-metal/csi-driver-image/assets/34534103/46d8c051-63ed-4fbf-b4c4-39ca7acdf612">